grafana/dashboards/cmode/nfs_clients.json

```diff
@@ -99,7 +99,7 @@
       },
       "id": 29,
       "options": {
-        "content": "You must enable the Rest collector in your harvest.yml config and uncomment the nfs_clients.yaml section in your $HARVEST/conf/rest/default.yaml file. Please see https://github.com/NetApp/harvest/wiki/FAQ#nfs-clients-dashboard for more details",
+        "content": "You must enable the Rest collector in your harvest.yml config and uncomment the nfs_clients.yaml section in your $HARVEST/conf/rest/default.yaml file. Please see https://github.com/NetApp/harvest/wiki/FAQ#nfs-clients-dashboard for more details. Enabling nfs_clients.yaml may slow down data collection.",
```

Comment:
You must enable the Rest collector in your `harvest.yml` config and uncomment the `nfs_clients.yaml` section in your `$HARVEST/conf/rest/default.yaml` file. Please see [nfs clients dashboard](https://github.com/NetApp/harvest/wiki/FAQ#nfs-clients-dashboard) for more details. Enabling `nfs_clients.yaml` may slow down data collection.